### PR TITLE
docs: Add brew, curl, unzip as pre-req

### PR DESCRIPTION
**Pull request by vburckhardt**
### Description

Brew is missing as pre-req in documentation

**Check the relevant boxes:**
- [ ] Bug fix (nonbreaking change that fixes an issue)
- [ ] New feature (nonbreaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI related update (pipeline, etc.)

### Checklist

- [ ] If relevant, a test for the change has been added or updated as part of this PR.
- [ ] If relevant, documentation for the change has been added or updated as part of this PR.

### Merge

- Merge using "Squash and merge".
- Make sure to use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents. The commit message determines whether a new version of the modules needs to be released, and if so, which semver number to use).

### CODE REVIEW

File: docs/local-dev-setup.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Follow these steps to set up your local development environment for working with
 - Make sure that you have a current version of Python 3 and pip.
 - Make sure that a recent stable version of [Go is installed](https://go.dev/doc/install) and available in your PATH environment variable.
 - Make sure the [Brew](https://brew.sh) is installed
+- Make sure that [curl](https://curl.se/) and unzip are installed.
```
Comment (SirSpidey) [inline docs/local-dev-setup.md:13]: @vburckhardt We don't have anything in the docs that's mentions curl. What process needs that? (Also requires an install only for Windows.)

Also no mention of extracting files. What do we need an unzip tool for (also already included by default for windows and macOS).

**Review by SirSpidey — COMMENTED** [review]
questions about why and where we need these tools. If there's a prereq, we should reference its use.

File: docs/local-dev-setup.md (hunk 1 above)
Comment (ocofaigh) [reply docs/local-dev-setup.md:13]: Some confusion here I think..
We dont need curl or unzip for development. However they are needed to run the migration script that is in this PR https://github.com/terraform-ibm-modules/common-dev-assets/pull/85

So I think we should remove this line. Brew is valid right now, as we install checkov with brew

Comment (SirSpidey) [reply docs/local-dev-setup.md:13]: Thanks @ocofaigh. Looks like that migration script renders redundant most of the migration content I just pushed in #12.

Comment (vburckhardt) [reply docs/local-dev-setup.md:13]: @ocofaigh - curl and unzip are needed by install-deps.sh which is part of the regular dev workflow, so these are valid here.

Comment (ocofaigh) [reply docs/local-dev-setup.md:13]: oh yes, valid point!

File: docs/local-dev-setup.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ Follow these steps to set up your local development environment for working with
 - On Microsoft Windows, set up Windows Subsystem for Linux [WSL](https://ubuntu.com/wsl) and run commands within WSL.
 - Make sure that you have a current version of Python 3 and pip.
 - Make sure that a recent stable version of [Go is installed](https://go.dev/doc/install) and available in your PATH environment variable.
+- Make sure the [Brew](https://brew.sh) is installed
+- Make sure that [curl](https://curl.se/) and unzip are installed.
```
Comment (SirSpidey) [inline docs/local-dev-setup.md:13]:
```suggestion
- Make sure that [curl](https://curl.se/) and a .zip file tool are installed.
```

**Review by SirSpidey — COMMENTED** [review]
Requested a change to not use a specific .zip file tool. Feel free to merge this without my further approval.

**Review by SirSpidey — APPROVED** [review]
👍

**Pull request merged**